src/inc/msquic.h

```diff
@@ -1096,11 +1098,20 @@ typedef struct QUIC_LISTENER_EVENT {
             BOOLEAN AppCloseInProgress  : 1;
             BOOLEAN RESERVED            : 7;
         } STOP_COMPLETE;
+        struct {
+            BOOLEAN DosModeEnabled : 1;
+            BOOLEAN RESERVED       : 7;
+        } DOS_MODE;
```

Comment:
```suggestion
        } DOS_MODE_CHANGED;
```
Needs to match the event type name.

Reply:
Done.